webpack.common.config.js

```diff
@@ -64,6 +64,14 @@ var workerConfig = function() {
     }
 };
 
+try {
+    // eslint-disable-next-line global-require, import/no-unresolved
+    xmoduleJS = require('./common/static/xmodule/webpack.xmodule.config.js');
+} catch (error) {
+    xmoduleJS = {};
```

Comment:
Can you please add a comment explaining this? I don't understand why we'd want to suppress errors on the import of what seems to be an important module. If the user has somehow skipped generating the XModule JS, then this file will be missing and presumably things will break, so wouldn't they need to be warned?

Reply:
It's been so long that I can't fully remember but I've mentioned the reason here: https://github.com/edx/edx-platform/pull/19986#issuecomment-515693033

It could be that this issue no longer exists, I will try reverting this change and see if I still get errors.

Reply:
Yup, it's still an issue. I've reintroduced this change and will add a comment.

Reply:
Thanks. Still seems a little strange to me though. Wouldn't it be better to configure CI to generate the xmodule compiled files before running, rather than suppressing errors like this?

Reply:
Btw I can't seem to see where to see the error you're mentioning (GitHub is suppressing the force push details I think?) so I just pushed a temp commit to take a look at it, since I have a couple hours I can use and I'd like to understand it before merging.

Reply:
@xitij2000 Ok, I found you a cleaner workaround, thanks to noticing a similar pattern in [this file](https://github.com/edx/edx-platform/blob/61f8df54532c8c949f9caf9bf8c3959e4223f954/openedx/features/course_experience/static/course_experience/js/WelcomeMessage.js#L3-L4).

Please review and if it looks good, squash my commits into yours and then I'll approve and merge.

Reply:
@bradenmacdonald The change looks OK, I think I tried something similar but it didn't work because I was suppressing specific errors instead of just disabling it entirely for that line. I've squashed the commits and this is good to merge.